### PR TITLE
Pass in a persistence model for saving

### PR DESCRIPTION
**Pull request by Flaque**
# Changes made
* Decouple the saving from the rest of the Feed. Instead it's passed into the feed, which lets us test the feed without actually creating files and therefore causing issues :( 

New example of a feed
``` js 
<Feed persist={someObjectContainingPersitenceFunctions} /> 
```

* This fixes both the bug not allowing us to test on storybook and the bug that adds extra copies of "hey I have some text" to our dev env's.

### CODE REVIEW

File: packages/aurora-feed/test/feed.test.js [hunk 1]
```diff
@@ -3,24 +3,37 @@ import { shallow, mount } from "enzyme";
 import Feed from "../index.js";
 import { EditorState, ContentState } from "draft-js";
 
+/**
```
Comment (kmccrohan) [inline packages/aurora-feed/test/feed.test.js]: very creative way of faking saving.

File: stories/feed.js [hunk 2]
```diff
@@ -3,11 +3,24 @@ import Feed from "../packages/aurora-feed";
 import { Background } from "../packages/aurora-ui";
 import { Themed } from "../packages/aurora-theme";
 
+/**
```
Comment (kmccrohan) [inline stories/feed.js]: This is duplicated from another file. Could we avoid this?

Comment (Flaque) [reply stories/feed.js]: It's fine to duplicate in stories I think, they're just docs.

File: packages/aurora-feed/test/feed.test.js (hunk 1 above)
Comment (Flaque) [reply packages/aurora-feed/test/feed.test.js]: 💅💅💅

File: stories/feed.js (hunk 2 above)
Comment (Flaque) [reply stories/feed.js]: Actually, maybe I'm wrong. I have an idea.

Comment (Flaque) [reply stories/feed.js]: Okay, I moved these into `aurora-persist`

**Pull request merged**